docs/topics/db/queries.txt

```diff
@@ -530,55 +530,74 @@ those latter objects, you could write::
-select all blogs that contain entries with both *"Lennon"* in the headline
-and that were published in 2008 (the same entry satisfying both conditions),
-we would write::
+When spanning a :class:`~django.db.models.ManyToManyField` or a reverse
+:class:`~django.db.models.ForeignKey`, such as from ``Blog`` to ``Entry``,
+filtering on multiple attributes raises the question of whether to require each
+attribute to coincide in the same related object. We might seek blogs that have
+an entry from 2008 with *“Lennon”* in its headline, or we might seek blogs that
+merely have any entry from 2008 as well as some newer or older entry with
+*“Lennon”* in its headline.
+
+To select all blogs containing at least one entry from 2008 having
+*"Lennon"* in its headline (the same entry satisfying both conditions), we
+would write::
 
     Blog.objects.filter(entry__headline__contains='Lennon', entry__pub_date__year=2008)
 
-To select all blogs that contain an entry with *"Lennon"* in the headline
-**as well as** an entry that was published in 2008, we would write::
+Otherwise, to perform a more permissive query selecting any blogs with merely
+*some* entry with *"Lennon"* in its headline and some *other* entry from 2008,
+we would write::
 
     Blog.objects.filter(entry__headline__contains='Lennon').filter(entry__pub_date__year=2008)
 
 Suppose there is only one blog that had both entries containing *"Lennon"* and
 entries from 2008, but that none of the entries from 2008 contained *"Lennon"*.
 The first query would not return any blogs, but the second query would return
-that one blog.
-
-In the second example, the first filter restricts the queryset to all those
-blogs linked to entries with *"Lennon"* in the headline. The second filter
-restricts the set of blogs *further* to those that are also linked to entries
-that were published in 2008. The entries selected by the second filter may or
-may not be the same as the entries in the first filter. We are filtering the
-``Blog`` items with each filter statement, not the ``Entry`` items.
+that one blog. (This is because the entries selected by the second filter may
+or may not be the same as the entries in the first filter. We are filtering the
+``Blog`` items with each filter statement, not the ``Entry`` items.) In short,
+if each condition needs to match the same related object, then each should be 
+contained in a single :meth:`~django.db.models.query.QuerySet.filter` call.
+
+.. note::
+
+    As the second (more permissive) query chains multiple filters, it performs
+    multiple joins to the primary model, potentially yielding duplicates.
+
+        >>> from datetime import date
+        >>> beatles = Blog.objects.create(title='Beatles Blog')
+        >>> pop = Blog.objects.create(title='Pop Music Blog')
+        >>> a1 = Entry.objects.create(
+        ...     blog=beatles,
+        ...     headline='New Lennon Biography',
+        ...     pub_date=date(2008, 6, 1)
+        ... )
```

Comment:
I noticed that most of `Entry`'s fields are required and don't have default values so `create()` will raise:
```
django.db.utils.IntegrityError: NOT NULL constraint failed: ...
```
I added a separate commit with default values.

Reply:
Terrific, thanks!